.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
```

Comment:
If I understand correctly, we'd like feature requests to currently go to forums. Should we give the feature_request issue template something similar to the same treatment as eslint does for issues that they want to go elsewhere? 

EG: https://github.com/eslint/eslint/issues/new/choose, notice the treatment for 'Question' and 'Security Issue'